src/amo/components/Search/styles.scss

```diff
@@ -45,7 +45,7 @@
   .SearchResults {
     @include respond-to(large) {
       grid-column: 2;
-      grid-row: 2 / 4;
+      grid-row: 2 / 5;
```

Comment:
Do we need to specify specific rows for the layout at all? I wonder if we could stop doing that where we can rely on implicit ordering.

Reply:
Not sure we can because of the difference between mobile and desktop pages.

Reply:
We can look at that separately, I might be wrong (I'm still getting used to grids) but I think aside from intricate layouts specific rows might be something that could be factored out.